src/cases/cases.module.ts

```diff
@@ -23,22 +23,32 @@ import {
   SearchResultModule,
   CreateCaseFiltersModule,
   CaseListFiltersModule,
-  AlertService
+  AlertService,
+  EventLogModule,
+  CaseReferencePipe,
+  ActivityModule,
+  CaseHeaderModule,
+  ConditionalShowModule,
+  LabelSubstitutorModule,
+  MarkdownModule,
+  PaletteModule,
+  FieldsPurger,
+  NavigationNotifierService
```

Comment:
You've added quite a few modules here - are all of these required for the Payments component?